src/components/Directory/List.js

```diff
@@ -32,6 +32,13 @@ tools
 
 
 export const list = [
+  {
+    name: "Towards Liberty",
+    url: "https://towardsliberty.com/",
+    type: "education",
+    description:
+      "The Agora Towards Liberty is a gathering place for peers to foster a holistic understanding of applied freedom strategies with tailor made education for your individual curiosity. Using the ancient Socratic method of truth discovery through reasoning and gathering with peers to discuss in depth any topic worth understanding."
```

Comment:
```suggestion
      "The Agora Towards Liberty is a gathering place for peers to foster a holistic understanding of applied freedom strategies with tailor made education for your individual curiosity. Using the ancient Socratic method of truth discovery through reasoning and gathering with peers to discuss in depth any topic worth understanding. Also reachable through Tor at http://2wp7zwpx6myz7kbdjuthmtccvs6p3tildwhrnm4zmv2fnizrfeo7hvyd.onion/"
```

For the onion link, either this can work (though the Tor link won't be clickable), or simply adding a second entry with the Tor address instead of clearnet.
Your choice.

Reply:
this is the more reasonable solution, thanks.